packages/react/src/components/Accordion/AccordionItem.js

```diff
@@ -84,11 +85,13 @@ function AccordionItem({
         onKeyDown={onKeyDown}
         type="button">
         <ChevronRight16 className={`${prefix}--accordion__arrow`} />
-        <div className={`${prefix}--accordion__title`}>{title}</div>
+        <Text as="div" className={`${prefix}--accordion__title`}>
+          {title}
+        </Text>
       </Expand>
-      <div id={id} className={`${prefix}--accordion__content`}>
+      <Text as="div" id={id} className={`${prefix}--accordion__content`}>
```

Comment:
I think we'll have to keep this as a `div` since the content could contain things other than text nodes, potentially

Reply:
I think specifically that we should keep this as before instead of changing it to `Text`. Sorry that was unclear!